tests/ConfigurationServiceFile/errors.test.ts

```
@@ -2,28 +2,18 @@ import { ConfigurationService } from 'api/ConfigurationService';
 import { ConfigurationServiceFile } from 'provider/FileProvider';
 import { messages } from '../../src/utils';
 
-const filename = 'test.conf.json';
-const repository = 'Adele';
+const token = 'test';
+const repository = 'repository.json';
+const notFoundRepository = 'Freddie Mercury';
 const key = 'Hello';
-const configService = new ConfigurationServiceFile(filename);
+const notFoundKey = 'Goodbye';
+const configService = new ConfigurationServiceFile(token);
 
 describe('Test suite for the ConfigurationServiceFile', () => {
-  
-  it('New instance of service should throw \'filenameNotProvided\' error', async () => {
-    expect.assertions(1);
-    expect(() => new ConfigurationServiceFile()).toThrow(new Error(messages.filenameNotProvided));
-  });
-
-  it('New instance of service should throw \'fileOrDirectoryNotExist\' error', async () => {
-    expect.assertions(1);
-    expect(() => new ConfigurationServiceFile('test.conf.not.exist.json'))
-      .toThrow(new Error(messages.fileOrDirectoryNotExist));
-  });
 
-  it('New instance of service should throw \'fileNotValid\' error', async () => {
+  it('New instance of service should throw \'tokenNotProvided\' error', async () => {
     expect.assertions(1);
-    expect(() => new ConfigurationServiceFile('test.conf.not.valid.json'))
-      .toThrow(new Error(messages.fileNotValid));
+    expect(() => new ConfigurationServiceFile()).toThrow(new Error(messages.tokenNotProvided));
```

have you checked silent swallowing of errors? I think we have to return expect

---

I will return